### PR TITLE
Make client calls

### PR DESCRIPTION
**Pull request by shea-maykinmedia**
Solution for https://github.com/maykinmedia/open-personen/issues/93

This pull request does a few things:
1. Removes the mocking of responses so we now handle the actual response from a client
2. Handles two different formats to XML
3. Ensure an exception isn't thrown if an attribute is missing and now fills in a default value

### CODE REVIEW

File: src/openpersonen/api/tests/response_templates/ResponseNatuurlijkPersoonSoapUI.xml [hunk 1]
```diff
@@ -0,0 +1,140 @@
+<env:Envelope xmlns:env="http://schemas.xmlsoap.org/soap/envelope/" xmlns:wsa="http://www.w3.org/2005/08/addressing">
```
Comment (joeribekker) [inline src/openpersonen/api/tests/response_templates/ResponseNatuurlijkPersoonSoapUI.xml:1]: Why is this file here? Unlike the others, I don't see this one used (in this PR)

File: src/openpersonen/api/tests/views/test_ingeschrevenpersoon.py [hunk 2]
```diff
@@ -32,30 +32,29 @@ class TestIngeschrevenPersoon(APITestCase):
     def setUp(self):
         super().setUp()
         self.url = StufBGClient.get_solo().url
+        self.user = User.objects.create(username="test")
+        self.token = Token.objects.create(user=self.user)
```
Comment (joeribekker) [inline src/openpersonen/api/tests/views/test_ingeschrevenpersoon.py:36]: Please create factories. You can use FactoryBoy for this

See for example here: https://github.com/open-zaak/open-zaak/blob/master/src/openzaak/components/autorisaties/tests/factories.py

Doing something similar can result in:

```
self.token = TokenFactory.create()
self.user = self.token.user  # Automatically created by the TokenFactory.
```

**Review by joeribekker — CHANGES_REQUESTED** [review]
Nice work on the test cases and mocking responses.

Really minor issues here. I saw another file ending in SoapUI. not sure if that one is used somewhere.

File: src/openpersonen/api/tests/response_templates/ResponseNatuurlijkPersoonSoapUI.xml (hunk 1 above)
Comment (shea-maykinmedia) [reply src/openpersonen/api/tests/response_templates/ResponseNatuurlijkPersoonSoapUI.xml:1]: I think this was initially used for some API testing but I don't think it's being used now so I'll remove it and remove any other files that are no longer being used.

File: src/openpersonen/api/tests/views/test_ingeschrevenpersoon.py (hunk 2 above)
Comment (shea-maykinmedia) [reply src/openpersonen/api/tests/views/test_ingeschrevenpersoon.py:36]: Will do!

**Review by joeribekker — APPROVED** [review]
